scripts/hid_configurator/README.rst

```diff
@@ -56,8 +56,13 @@ Complete the following steps:
 #. Select the appropriate HIDADPI library version for the used Windows system (either ``x86`` or ``x64``).
    Copy the :file:`hidapi.dll` file and paste it into either of the following directory:
 
-   * The directory where used Python executable is located.
-   * The :file:`Windows\\System32` directory, for example :file:`C:\\Windows\\System32`.
+   * The directory hid_configurator directory.
+   * Any folder you want, and add the folder path as an argument to the script:
+
+      .. parsed-literal::
+      :class: highlight
+
+      python3 configurator_cli.py --dll_path "C:\\Users\\user\\hidapi-win\\x64" show
```

Comment:
Discussed offline. We will desist from making the dll path configurable from CLI argument (it significantly complicates implementation and causes some compliance issues while not brining much added value).